app/templates/components/navigation-bar.hbs

```diff
@@ -0,0 +1,44 @@
+<nav class="navbar navbar-light">
+  <div class="container">
+    <a class="navbar-brand" href="index.html">conduit</a>
```

Comment:
While you're here 😬Could we fix the logo link to navigate to `home`? There's an issue for it too #41